ts/features/payments/bizEventsTransaction/screens/PaymentsTransactionBizEventsDetailsScreen.tsx

```diff
@@ -83,6 +88,14 @@ const PaymentsTransactionBizEventsDetailsScreen = () => {
   };
 
   const handleOnDownloadPdfReceiptError = () => {
+    analytics.trackPaymentsDownloadReceiptError({
+      organization_name: paymentAnalyticsData?.receiptOrganizationName,
+      first_time_opening: paymentAnalyticsData?.receiptFirstTimeOpening,
+      user:
+        userFiscalCode === paymentAnalyticsData?.receiptPayerFiscalCode
+          ? "payer"
+          : "payee"
```

Comment:
Could it make senso to have an utility function for this kind of ternary operation? What do you think?

Reply:
It makes sense, however, I noticed that we can get this information from the API that returns the list of transactions before accessing the detail. It returns us the `isPayer` flag, so by doing that we don't even need to have to take the user's information such as social security number and compare it.

Take a look at what was implemented in the following commit: a85819a72292a7f6623008dd91e5a94dcb9a0f1f and let me know what do you think about.

Reply:
Good catch!